### DISCUSSION TITLE
2 wire:model on single input not working

### DISCUSSION THREAD

**Discussion by abdulrehman-dar** [livewire/livewire#4466] [7. Bugs (archived)]
I'm using 2 wire:model on a single input in which one wire:model is not working the first one always work but 2nd don't work.
The first one I'm using for the input value is if the input value changes it will send the request and the number formator under input work. 
The 2nd wire:model is to use an updated hook so I can calculate the commission on the base of the first input.
If an input has two Wire model the first one always work not the 2nd one.
```
                 <div class="mb-1">
                      <label>{{__('Agent Commission')}}</label>
                      <input type="number" wire:model="agent_commission" wire:model="commission">                                                                                                                                              <label>
                      @if(!empty($agent_commission))
                      <?php $f = new NumberFormatter("en-IN", NumberFormatter::SPELLOUT);
                       echo $f->format($agent_commission)
                      ?>
                      @endif
                      </label>
                      </div>

**Comment by abdulrehman-dar** [comment]
to solve this issue The hack is use wire:model.debounce on anyone
```
<input type="number" wire:model.debounce.100ms="commission" wire:model="agency_commission">
```

**Comment by joshhanley** [comment]
@abdulrehman-dar Livewire doesn't support using two `wire:model` on once input. Can you explain a bit as to why you need this instead of just using the one `wire:model` and calculating both commissions in the updated hook for the one property?

Thanks!
